meteor/lib/api/triggers/actionFactory.ts

```diff
@@ -476,6 +487,9 @@ export function createAction(action: SomeAction, showStyleBase: ShowStyleBase):
 				}
 			}
 		case PlayoutActions.deactivateRundownPlaylist:
+			if (Meteor.isClient && action.filterChain.every((link) => link.object === 'view')) {
```

Comment:
What does this check mean and why does "link.object === 'view'" has anything to do with us showing a modal?
Would be nice to extract it into a nicely named function :)

Reply:
I'm not sure. My guess is, it's to make sure the action was triggered from the ui